honda_crv_ex_2017_can.dbc

```diff
@@ -109,73 +116,85 @@ BO_ 401 GEARBOX: 8 PCM
  SG_ GEAR : 39|8@0+ (1,0) [0|255] "" XXX
  SG_ ZEROS_BOH : 47|2@0+ (1,0) [0|3] "" XXX
  SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|15] "" NEO
 
 BO_ 420 VSA_STATUS: 8 VSA
  SG_ ESP_DISABLED : 28|1@0+ (1,0) [0|1] "" NEO
  SG_ USER_BRAKE : 7|16@0+ (0.015625,-103) [0|1000] "" NEO
  SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" NEO
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|15] "" NEO
 
 BO_ 427 XXX_5: 3 VSA
+ SG_ COUNTER : 21|2@0+ (1,0) [0|3] "" XXX
+ SG_ CHECKSUM : 19|4@0+ (1,0) [0|15] "" XXX
 
 BO_ 432 STANDSTILL: 7 VSA
  SG_ WHEELS_MOVING : 12|1@0+ (1,0) [0|1] "" NEO
  SG_ BRAKE_ERROR_1 : 11|1@0+ (1,0) [0|1] "" NEO
  SG_ BRAKE_ERROR_2 : 9|1@0+ (1,0) [0|1] "" NEO
  SG_ COUNTER : 53|2@0+ (1,0) [0|3] "" NEO
- SG_ CHECKSUM : 51|4@0+ (1,0) [0|3] "" NEO
+ SG_ CHECKSUM : 51|4@0+ (1,0) [0|15] "" NEO
 
 BO_ 441 XXX_6: 5 VSA
+ SG_ COUNTER : 37|2@0+ (1,0) [0|3] "" XXX
+ SG_ CHECKSUM : 35|4@0+ (1,0) [0|15] "" XXX
 
 BO_ 446 XXX_7: 3 VSA
+ SG_ COUNTER : 21|2@0+ (1,0) [0|3] "" XXX
+ SG_ CHECKSUM : 19|4@0+ (1,0) [0|15] "" XXX
 
 BO_ 450 ELECTRONIC_PARKING_BRAKE: 8 EPB
  SG_ EPB_ACTIVE : 3|1@0+ (1,0) [0|1] "" NEO
  SG_ EPB_STATE : 29|2@0+ (1,0) [0|3] "" NEO
- SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" BDY
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" BDY
+ SG_ COUNTER : 61|2@0+ (1,0) [0|3] "" XXX
+ SG_ CHECKSUM : 59|4@0+ (1,0) [0|15] "" XXX
 
 BO_ 464 WHEEL_SPEEDS: 8 VSA
- SG_ WHEEL_SPEED_FL : 7|15@0+ (0.002759506,0) [0|70] "m/s" NEO
- SG_ WHEEL_SPEED_FR : 8|15@0+ (0.002759506,0) [0|70] "m/s" NEO
- SG_ WHEEL_SPEED_RL : 25|15@0+ (0.002759506,0) [0|70] "m/s" NEO
- SG_ WHEEL_SPEED_RR : 42|15@0+ (0.002759506,0) [0|70] "m/s" NEO
- SG_ CHECKSUM : 59|4@0+ (1,0) [0|3] "" NEO
+ SG_ WHEEL_SPEED_FR : 8|15@0+ (0.00286,0) [0|70] "m/s" NEO
+ SG_ WHEEL_SPEED_RL : 25|15@0+ (0.00286,0) [0|70] "m/s" NEO
+ SG_ WHEEL_SPEED_RR : 42|15@0+ (0.00286,0) [0|70] "m/s" NEO
+ SG_ WHEEL_SPEED_FL : 7|15@0+ (0.00286,0) [0|70] "m/s" NEO
```

Comment:
how are you finding the factor? trying to match some gps speed or comparing with other speed signals?

Reply:
Comparing with dash speed, it was off by about 1-3 miles an hour... Any recommendations on getting this exact?

Reply:
Also, wouldn't it be better to have this value the same across hondas and use a multiplier based on wheel size in openpilot?

Reply:
yeah, dash speed is always fictitiously increased (by law, so people drive at speed limits conservatively)
I'm sure the conversion factor (wheel radius) is probably already done somewhere in the honda code, otherwise the discrepancy would be bigger: aprrox tire diam for honda civic is 25.5", while for CRV is 29.1". If you want this speed to have physical meaning, I would probably recommend to keep the factor as it was. Not a big deal either way... just FYI

Reply:
ok thanks, in that case, i'll revert this in the next update I send. I will look for somewhere else to adjust this factor to match the dash since its a little confusing when setting at a value and not seeing it reflected on neo speed, especially when relying on stock ACC.